Source/CloudyPanelPlugin/Private/CloudyPanelPlugin.cpp

```diff
@@ -13,41 +17,20 @@ DEFINE_LOG_CATEGORY(ModuleLog)
 #define IP "127.0.0.1"
```

Comment:
Perhaps you could simplify the definition of an `endpoint` or `address` if you do something like:

    #define MY_SERVER_ENDPOINT FIPv4Endpoint(FIPv4Address(0, 0, 0, 0), 55556)

https://github.com/insert-coin/RemoteController/blob/master/RemoteController/Source/RemoteController/Private/RemoteControllerPCH.h#L3